source/sdk/lang/types.ooc

```diff
@@ -122,8 +122,10 @@ Closure: cover {
 /** An object storing a value and its class. */
 Cell: class <T> {
     val: __onheap__ T
+    type: Class = null
 
     init: func(=val)
+    init: func ~withtype(=val, =type)
```

Comment:
I don't think we should add this just to get `HashDictionary` (which we're not even using) working. Besides, `Cell` should already know what `T` is, it shouldn't need to be stored like this. I'm going to let this sit for a while until someone comes up with a better idea.